src/states/intro.js

```diff
@@ -0,0 +1,93 @@
+import Phaser from 'phaser';
+import config from '../config';
+
+const textHoverColor = '#555555';
```

Comment:
Maybe we should also put this hover color into the config?

Reply:
Yeah, good idea. I'll move it and update the PR.

As for the help keys: I think they should be available during the game